.github/workflows/tests_sdk_cpp.yml

```diff
@@ -0,0 +1,39 @@
+name: C++ SDK test
+on: [workflow_call]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.8"
```

Comment:
3.8 is EOL, 3.10 would allow more runway

Reply:
Good catch - changed